target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,6 +138,8 @@ case "$FIRMWARE" in
 	archer-c59-v1|\
 	archer-c60-v1)
 		ath10kcal_extract "art" 20480 12064
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
I thought about it. I sent my boarddata_0.bin from OEM firmware to kvalo and he did not answer.
I create PR on his repo: https://github.com/kvalo/ath10k-firmware/pull/5 it board-2.bin with my boarddata_0.bin.
This board-2.bin work OK wihout any board.bin ( pre_cal_data is still needed)

>  +	$(INSTALL_DATA) \
>  +		$(PKG_BUILD_DIR)/QCA9888/hw2.0/board-2.bin \
>  +		$(1)/lib/firmware/ath10k/QCA9888/hw2.0/board.bin 

i dont know if it is nedded for another board. On QCA9886 is redundant.